fuzzers/run_fuzzer.py

```diff
@@ -313,7 +314,7 @@ def mem_convert(s):
         if s.endswith(u):
             break
 
-    v = float(s[:-len(u)])
+    v = float(s[:-len(u)].replace(',', '.'))
```

Comment:
Can you put a small comment what is going on here?

Reply:
That's the traceback I get:
```
2021-01-11T18:55:34 - xc7a200tffg1156-1/001-part-yaml             -   10s: Traceback (most recent call last):
2021-01-11T18:55:34 - xc7a200tffg1156-1/001-part-yaml             -   10s:   File ".//run_fuzzer.py", line 564, in run_fuzzer
2021-01-11T18:55:34 - xc7a200tffg1156-1/001-part-yaml             -   10s:     mem = get_memory()['mem']
2021-01-11T18:55:34 - xc7a200tffg1156-1/001-part-yaml             -   10s:   File ".//run_fuzzer.py", line 356, in get_memory
2021-01-11T18:55:34 - xc7a200tffg1156-1/001-part-yaml             -   10s:     d[k] = mem_convert(v)
2021-01-11T18:55:34 - xc7a200tffg1156-1/001-part-yaml             -   10s:   File ".//run_fuzzer.py", line 316, in mem_convert
2021-01-11T18:55:34 - xc7a200tffg1156-1/001-part-yaml             -   10s:     v = float(s[:-len(u)])
2021-01-11T18:55:34 - xc7a200tffg1156-1/001-part-yaml             -   10s: ValueError: could not convert string to float: '5,4'
2021-01-11T18:55:35 - xc7a200tffg1156-1/001-part-yaml             -   11s: Warning: Killed program which should have been dead!
2021-01-11T18:55:35 - xc7a200tffg1156-1/001-part-yaml             -   11s: Finishing (User:0s System:0s).

```
and my free results:

```
daniel@desktop-daniel:~/work/symbiflow/prjxray$ free -mh
              total        used        free      shared  buff/cache   available
Mem:           31Gi       5,0Gi        20Gi       222Mi       6,0Gi        25Gi
Swap:         979Mi          0B       979Mi

```
Seems like my machine is still partly configured with German output, which outputs commas instead of dots.

Reply:
Sure, please add a comment.  The other option here is to override the LOCALE to avoid this.

Reply:
I highly recommend we explicitly set the `LOCALE` value, maybe somewhere like in the `Makefile`?